newdle/api.py

```diff
@@ -166,6 +167,18 @@ def get_newdle(code):
     return schema_cls().jsonify(newdle)
 
 
+@api.route('/newdle/<code>', methods=('PATCH',))
+@use_args(UpdateNewdleSchema(), locations=('json',))
+def update_newdle(args, code):
+    newdle = Newdle.query.filter_by(code=code).first_or_404('Invalid code')
+    if newdle.creator_uid != g.user['uid']:
+        return jsonify(error='forbidden'), 403
```

Comment:
you can just `raise Forbidden` here - this will already result in a jsonified error with the default message of the error (which is fine here, since it should never happen anyway)